doc/plugins.dox

```diff
@@ -170,14 +171,22 @@ Geany's website at http://www.geany.org/Support/BuildingOnWin32.
 
 @section helloworld "Hello World"
 
-When writing a plugin, you will find a couple of functions or macros which are mandatory
-and some which are free to use for implementing some useful feature once your plugin
-becomes more powerful like including a configuration or help dialog.
+@note This section describes the new entry points for plugins introduced with Geany 1.26. A short
+summary of the legacy entry points is given at page @ but it should not be used any more.
+
+When writing a plugin, you will find a couple of functions which are mandatory and some which are
+free to use for implementing some useful feature once your plugin becomes more powerful like
+including a configuration or help dialog.
+
+@subsection beginning Beginnings of any Plugin
 
 You should start your plugin with including some of the needed C header files and defining
 some basic global variables which will help you to access all needed functions of the plugin
 API in a more comfortable way.
```

Comment:
@kugel- I know its an existing sentence, but its very strained English, suggest:

"The plugin should start with including the required header files and defining the global variables which allow accessing the functions of the plugin API more easily."

Is this still the case? You seem to have deleted the description of these below.